mesonbuild/backend/vs2010backend.py

```diff
@@ -550,19 +550,26 @@ def create_basic_crap(self, target, guid):
 
     def gen_run_target_vcxproj(self, target, ofname, guid):
         root = self.create_basic_crap(target, guid)
+        depend_files = self.get_custom_target_depend_files(target)
+
         if not target.command:
-            # FIXME: This is an alias target that doesn't run any command, there
-            # is probably a better way than running a this dummy command.
-            cmd_raw = python_command + ['-c', 'exit']
+            # This is an alias target and thus doesn't run any command. It's
+            # enough to emit the references to the other projects for them to
+            # be built/run/..., if necessary.
+            assert(isinstance(target, build.AliasTarget))
+            assert(len(depend_files) == 0)
         else:
+            assert(not isinstance(target, build.AliasTarget))
+
             _, _, cmd_raw = self.eval_custom_target_command(target)
-        depend_files = self.get_custom_target_depend_files(target)
-        target_env = self.get_run_target_env(target)
```

Comment:
I think you removed that line by mistake, but CI will hopefully tell :)

Reply:
Yea, sorry for that. I accidentally fixed it in 0.59 because I was bisecting and forgot a bisect reset. And then when rebasing I broke it :(